plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/attribute-filter/inspector.tsx

```diff
@@ -64,10 +64,12 @@ export const Inspector = ( {
 	);
 
 	if ( displayStyleOptions.length === 0 ) {
-		displayStyleOptions = getBlockTypes().filter( ( blockType ) =>
-			blockType.ancestor?.includes(
-				'woocommerce/product-filter-attribute'
-			)
+		displayStyleOptions = getBlockTypes().filter(
+			( blockType ) =>
+				blockType.name !== 'woocommerce/product-filter-clear-button' &&
```

Comment:
This makes me realize the current mechanism for registering display-style blocks is quite fragile, as blocks can be inner block of the filter block but not necessarily a display-style block.